config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,12 @@
+CarrierWave.configure do |config|
+  config.fog_provider = 'fog/aws'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.